pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -388,6 +388,10 @@ self: super: ({
   http-reverse-proxy = dontCheck super.http-reverse-proxy;
   servant-auth-server = dontCheck super.servant-auth-server;
 
+  sysinfo = overrideCabal (orig: {
+    doCheck = true;
```

Comment:
There are doCheck / dontCheck helpers in haskell.lib.

Also is this a bitflip? Don’t you want to disable the test?

Reply:
I think this should be `sysinfo = dontCheck super.sysinfo` here, as it used to be `doCheck = !…isDarwin`.